### ISSUE TITLE
msg parameter doesn't accept unicode strings

### ISSUE THREAD

**Issue by mmulder72** [astraw38/pytest-assume#6]
I have non-ascii characters in my test output, so I have to use unicode strings. However, when I pass a unicode string with non-ascii characters to `pytest.assume()` as the `msg` parameter, I get this error:
```python
    def assume(expr, msg=''):
        """
        Checks the expression, if it's false, add it to the
        list of failed assumptions. Also, add the locals at each failed
        assumption, if showlocals is set.
    
        :param expr: Expression to 'assert' on.
        :param msg: Message to display if the assertion fails.
        :return: None
        """
        if not expr:
            (frame, filename, line, funcname, contextlist) = inspect.stack()[1][0:5]
            # get filename, line, and context
            filename = os.path.relpath(filename)
            context = contextlist[0].lstrip() if not msg else msg
            # format entry
>           entry = "{filename}:{line}: AssumptionFailure\n\t{context}".format(**locals())
E           UnicodeEncodeError: 'ascii' codec can't encode character u'\xb5' in position 333: ordinal not in range(128)
```
I've found that a simple fix for this is to place a 'u' in front of the string where the `entry` variable is set like this:
```python
            entry = u"{filename}:{line}: AssumptionFailure\n\t{context}".format(**locals())
```

**Comment by astraw38** [comment]
Thanks for the report! 

I'll look into it & create some testcases to cover as well.

**Comment by astraw38** [comment]
Closed in #7 (sorry for delay on closing it), fix has been available in version 1.2.1
